packages/gcf-utils/package.json

```diff
@@ -39,8 +39,9 @@
     "gaxios": "^4.0.0",
     "get-stream": "^6.0.0",
     "into-stream": "^6.0.0",
+    "octokit-auth-probot": "^1.2.2",
     "pino": "^6.3.2",
-    "probot": "11.0.5",
+    "probot": "^11.0.5",
```

Comment:
I would suggest us keeping probot pinned.  They have proven to be untrustworthy with semver.

Reply:
I always forget this...thanks for keeping me honest :)